cli/index.ts

```diff
@@ -50,239 +358,50 @@ const compileProject = (
-                // don't recompile many times if we changed a lot of files
-                clearTimeout(timeoutID);
-              } else {
-                console.log("Watcher recompile project...");
-              }
-
-              timeoutID = setTimeout(() => {
-                clearTimeout(timeoutID);
-
-                if (!isCompiling) {
-                  // recompile project
-                  isCompiling = true;
-                  compileProject(projectDir, defaultSchemaOverride, assertionSchemaOverride).then(
-                    () => {
-                      console.log("Watcher ready for changes...");
-                      isCompiling = false;
-                    }
-                  );
-                }
-              }, RECOMPILE_DELAY);
-            });
-        }
-      });
-    }
-  )
-  .command(
-    "build [project-dir]",
-    "Build the dataform project. Produces JSON output describing the execution graph.",
-    yargs =>
-      addBuildYargs(yargs)
-        .positional("project-dir", {
-          describe: "The directory of the Dataform project.",
-          default: "."
-        })
-        .option("profile", {
-          describe: "The location of the profile JSON file to run against",
-          required: true
-        }),
-    argv => {
-      const profile = utils.readProfile(argv.profile);
-      const defaultSchemaOverride = !!argv["default-schema-override"]
-        ? path.resolve(argv["default-schema-override"])
-        : "";
-      const assertionSchemaOverride = !!argv["assertion-schema-override"]
-        ? path.resolve(argv["assertion-schema-override"])
-        : "";
-
-      compile({
-        projectDir: path.resolve(argv["project-dir"]),
-        defaultSchemaOverride,
-        assertionSchemaOverride
-      })
-        .then(graph => build(graph, parseBuildArgs(argv), profile))
-        .then(result => console.log(JSON.stringify(result, null, 4)))
-        .catch(e => console.log(e));
-    }
-  )
-  .command(
-    "run [project-dir]",
-    "Build and run the dataform project with the provided options.",
-    yargs =>
-      addBuildYargs(yargs)
-        .positional("project-dir", {
-          describe: "The directory of the Dataform project.",
-          default: "."
-        })
-        .option("profile", {
-          describe: "The location of the profile JSON file to run against",
-          required: true
-        })
-        .option("result-path", {
-          describe: "Path to save executed graph to JSON file",
-          type: "string"
-        }),
-    argv => {
-      console.log("Project status: starting...");
-      const profile = utils.readProfile(argv.profile);
-      const defaultSchemaOverride = !!argv["default-schema-override"]
-        ? path.resolve(argv["default-schema-override"])
-        : "";
-      const assertionSchemaOverride = !!argv["assertion-schema-override"]
-        ? path.resolve(argv["assertion-schema-override"])
-        : "";
+interface Cli {
```

Comment:
ICli - or shall we remove that lint rule?

Reply:
Done!